src/util/webextensionRPC.ts

```diff
@@ -447,3 +343,16 @@ export function getRemoteEventEmitter<EventType extends keyof RemoteEvents>(
     registerRemoteEventForwarder()
     return newEmitter
 }
+
+// Containing the evil globals here
+let rpcConnection: PortBasedRPCManager
+export const setRpcConnection = (sideName) => {
```

Comment:
Nitpick: I'd call this `initRpcConnection`, just to make clear it's doing a one-time operation and it's not expecting the caller to pass in an RPC connection. Also, since the rpcConnection object is an implementation detail that is not used except for one chained method call, you could just expose a function like `setupRpcConnection(options: { name: string, role: 'content' | 'background' })`